11ty/definitions/colonialism.md

```diff
@@ -1,5 +1,18 @@
 ---
 title: Colonialism
 slug: colonialism
-defined: false
+speech: noun
+excerpt: The act of enforcing one's political and ideological dominance in a country or region that is not of one's origin
+sub_terms:
+    - text: neo colonialism, western colonialism, modern colonialism
+defined: true
+reading:
+    - text: A brief context of colonialism
+      href: https://plato.stanford.edu/entries/colonialism/
 ---
+
+Colonialism is the act of forcing a group of people in a region or country to live under the policies and ideologies of an invading government or group of people for the purposes of asserting geo-political dominance. Though [often associated with imperialism](https://plato.stanford.edu/entries/colonialism/), the two words have different latin etymologies that drive how the connotation of colonialism and the word colony have evolved. The most notable instances of colonialism to the general public are western colonialism and recently, neo-colonialism.
+
+The practice of colonialism was most prominent with the British, Ottoman, and other European empires upon regions in the Americas, Asia, and Africa where slave trade was implemented as an economic vehicle. Contrary to popular belief, the colonialism also affected people of European descent, most notably, the Irish. Some studies have tried to [clarify the question](https://www.jstor.org/stable/20720270?seq=1) of whether or not this is true, but they tend to be written by non-native researchers, therefore muting a much needed context. Be mindful when researching this topic to note the origin of the researchers with respect to the colonies and colonizers they are writing about.
```

Comment:
I'm wondering here if we want "the slave trade" or "slave trade(s)" as I think right now the grammar reading smoothly?

Reply:
good point. I was mostly thinking about the trans Atlantic slave trade, but I am not a historian - the slave trade(s) covers everything. I'll change that